apps/law_memory/metrics.py

```
@@ -2,19 +2,39 @@
+    if law.shape != r_grid.shape:
+        raise ValueError("law must match r_grid")
+    return np.interp(query_r, r_grid, law, left=law[0], right=law[-1])
+
+
+def deformed_metric_profile(
+    r: np.ndarray, law: np.ndarray, xi: float = 0.382, lambda_h: float = 0.1457, r0: float = 1.0
+) -> Tuple[np.ndarray, np.ndarray]:
+    """Return the deformed ``phi`` and ``gamma`` profiles."""
 
     if r.size != law.size:
         raise ValueError("grid mismatch")
```

Ruff (0.13.3)

Avoid specifying long messages outside the exception class

(TRY003)